tests/testthat/test-uninstall.r

```diff
@@ -0,0 +1,23 @@
+context("Uninstall")
+
+test_that("uninstall() unloads and removes from library", {
+
+  # Make a temp lib directory to install test package into
+  old_libpaths <- .libPaths()
```

Comment:
I wonder if it would be useful to pull this out into `with_temp_lib` or something?